### PR TITLE
refactor(app): pUR FF Removal: Run Details Page

### LINKED ISSUE #8875: PUR FF Removal: Run Detail Page
All instances of preProtocolFlowWithoutRPC and their logic should be removed from the following locations within the run detail page.

```
opentrons/app/src/pages/Run/index.tsx:
  8  export function Run(): JSX.Element {
  9:   const isNewProtocolRunPage = useFeatureFlag('preProtocolFlowWithoutRPC')
  10  

opentrons/app/src/pages/Run/__tests__/Run.test.tsx:
  51      when(mockUseFeatureFlag)
  52:       .calledWith('preProtocolFlowWithoutRPC')
  53        .mockReturnValue(false)

  60      when(mockUseFeatureFlag)
  61:       .calledWith('preProtocolFlowWithoutRPC')
  62        .mockReturnValue(true)

opentrons/app/src/pages/Run/RunPanel/index.tsx:
  57  export function RunPanelComponent(props: Props): JSX.Element {
  58:   const isNewProtocolRunPanel = useFeatureFlag('preProtocolFlowWithoutRPC')
  59  

opentrons/app/src/pages/Run/RunPanel/__tests__/RunPanel.test.tsx:
  71      when(mockUseFeatureFlag)
  72:       .calledWith('preProtocolFlowWithoutRPC')
  73        .mockReturnValue(false)

  82      when(mockUseFeatureFlag)
  83:       .calledWith('preProtocolFlowWithoutRPC')
  84        .mockReturnValue(true)

opentrons/app/src/pages/Run/RunPanel/ModuleLiveStatusCards/StatusCard.tsx:
  28  export function StatusCard(props: Props): JSX.Element {
  29:   const isNewProtocolRunPanel = useFeatureFlag('preProtocolFlowWithoutRPC')
  30    const { t } = useTranslation('run_details')

opentrons/app/src/pages/Run/RunPanel/ModuleLiveStatusCards/__tests__/StatusCard.test.tsx:
  17      when(mockUseFeatureFlag)
  18:       .calledWith('preProtocolFlowWithoutRPC')
  19        .mockReturnValue(false)

  33      when(mockUseFeatureFlag)
  34:       .calledWith('preProtocolFlowWithoutRPC')
  35        .mockReturnValue(false)

opentrons/app/src/molecules/ModuleControls/TemperatureControl.tsx:
  47  }: Props): JSX.Element => {
  48:   const isNewProtocolRunPanel = useFeatureFlag('preProtocolFlowWithoutRPC')
  49    const [tempValue, setTempValue] = React.useState<string | null>(null)

opentrons/app/src/molecules/ModuleControls/TemperatureData.tsx:
  40  }: TemperatureDataProps): JSX.Element => {
  41:   const isNewProtocolRunPanel = useFeatureFlag('preProtocolFlowWithoutRPC')
  42    const { t } = useTranslation('run_details')



```

### PR DESCRIPTION
**Pull request by smb2268**
close #8875

# Overview

# Changelog

All instances of preProtocolFlowWithoutRPC and their logic should be removed from the following locations within the run detail page: 

- [ ] opentrons/app/src/pages/Run/index.tsx
- [ ] opentrons/app/src/pages/Run/__tests__/Run.test.tsx
- [ ] opentrons/app/src/pages/Run/RunPanel/index.tsx
- [ ] opentrons/app/src/pages/Run/RunPanel/__tests__/RunPanel.test.tsx
- [ ] opentrons/app/src/pages/Run/RunPanel/ModuleLiveStatusCards/StatusCard.tsx
- [ ] opentrons/app/src/pages/Run/RunPanel/ModuleLiveStatusCards/__tests__/StatusCard.test.tsx
- [ ] opentrons/app/src/molecules/ModuleControls/TemperatureControl.tsx
- [ ] opentrons/app/src/molecules/ModuleControls/TemperatureData.tsx

# Review requests
New run detail page should load whether or not FF is on

# Risk assessment

low

### CODE REVIEW

**Review by sakibh — APPROVED** [review]
LGTM! Tested it and the new Run Detail page shows up regardless of FF and it works just fine.

image

File: app/src/pages/Run/RunPanel/__tests__/RunPanel.test.tsx [hunk 1]
```diff
@@ -67,21 +49,7 @@ describe('RunSetupCard', () => {
...
-      .calledWith('preProtocolFlowWithoutRPC')
-      .mockReturnValue(false)
-    const { getByText } = render()
-    getByText('Mock Run Timer')
-    getByText('Mock Run Controls')
-    getByText('Mock Module Live Status Cards')
-    expect(mockRunTimeControl).not.toHaveBeenCalled()
-  })
-
   it('renders new run panel components when usePreProtocolWithoutRPC ff is set', () => {
```
Comment (b-cooper) [inline app/src/pages/Run/RunPanel/__tests__/RunPanel.test.tsx]:
```suggestion
  it('renders run panel components', () => {
```

File: app/src/pages/Run/RunPanel/index.tsx [hunk 2]
```diff
@@ -1,90 +1,14 @@
...
-import {
-  actions as robotActions,
-  selectors as robotSelectors,
-} from '../../../redux/robot'
-
-import { RunTimer } from './RunTimer'
-import { RunControls } from './RunControls'
 import { ModuleLiveStatusCards } from './ModuleLiveStatusCards'
 
-import type { MapDispatchToProps } from 'react-redux'
```
Comment (b-cooper) [inline app/src/pages/Run/RunPanel/index.tsx:17]: 💣

File: app/src/pages/Run/__tests__/Run.test.tsx [hunk 3]
```diff
@@ -47,19 +42,7 @@ describe('Run Page', () => {
...
-  it('renders legacy run page when usePreProtocolWithoutRPC ff is not set', () => {
-    when(mockUseFeatureFlag)
-      .calledWith('preProtocolFlowWithoutRPC')
-      .mockReturnValue(false)
-    const { getByText } = render()
-    getByText('Mock Run Log')
-    expect(mockRunDetails).not.toHaveBeenCalled()
-  })
-
   it('renders new run page when usePreProtocolWithoutRPC ff is set', () => {
```
Comment (b-cooper) [inline app/src/pages/Run/__tests__/Run.test.tsx]:
```suggestion
  it('renders run page', () => {
```

**Review by b-cooper — APPROVED** [review]
💥 looks good, just a couple of test names need updating

**Pull request merged**